bot.js

```diff
@@ -64,6 +64,15 @@ function matchPattern(p, w, pi, wi) {
 function saySomething() {
 	var message = document.getElementById('message').value;
 	var conversation = document.getElementById('conversation');
+
+	answer = reply(message);
```

Comment:
It would probably be better to add `var` here:

```
var answer = reply(message);
```

(But this is not really related to this class, just generally to JavaScript!)

Reply:
Oh yes - thanks, Tomas